tests/workbenches/utils.py

```diff
@@ -1,11 +1,23 @@
-from kubernetes.dynamic import DynamicClient, Resource, ResourceInstance
+from kubernetes.dynamic import DynamicClient
+from ocp_resources.self_subject_review import SelfSubjectReview
+from ocp_resources.user import User
+from simple_logger.logger import get_logger
 
+LOGGER = get_logger(name=__name__)
 
-def get_username(dyn_client: DynamicClient) -> str:
+
+def get_username(dyn_client: DynamicClient) -> str | None:
     """Gets the username for the client (see kubectl -v8 auth whoami)"""
-    self_subject_review_resource: Resource = dyn_client.resources.get(
-        api_version="authentication.k8s.io/v1", kind="SelfSubjectReview"
-    )
-    self_subject_review: ResourceInstance = dyn_client.create(self_subject_review_resource)
-    username: str = self_subject_review.status.userInfo.username
+    username: str | None
+    try:
+        self_subject_review = SelfSubjectReview(client=dyn_client, name="selfSubjectReview").create()
```

Comment:
Do you need to use wait=True?

Reply:
No, creating selfsubjectreview does not actually create anything in the cluster. The reply from the kube-apiserver is the only result.